### ISSUE TITLE
Reverse pattern and seq in examples - and specify which is which for the first!

### ISSUE THREAD

**Issue by maasha** [BIO-DIKU/SeqScan#46]
```
ATC =~ GAGATCGAG => 4,3,0,ATC
```

Should be

```
Seq          Pat
GAGATCGAG =~ ATC => 4,3,0,ATC
```

**Comment by RasmusFonseca** [comment]
Yes, pattern and match closer together. Wouldnt it make sense to avoid custom notations and go with 

```
//    Seq        Pat     Match
Match(GAGATCGAG, ATC) => 4,3,0,ATC
```

**Comment by maasha** [comment]
fixed with 9e324d5dde8a83c9a7bfe81f2fa6a895b5edbb1c
